src/st/st_debug.h

```diff
@@ -18,9 +18,11 @@ void EntityBackgroundBlock(Entity* self) {
         if (objInit->flags) {
             self->flags = objInit->flags;
         }
+#ifndef STAGE_IS_NO4
```

Comment:
This code should be shareable with this change in more places (BO4, etc.) but I have not done so for now